ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -203,9 +204,9 @@ export const NetworkListMenu = ({ onClose }) => {
               category: MetaMetricsEventCategory.Network,
               properties: {
                 location: 'Network Menu',
-                chain_id: currentChainId,
-                from_network: currentChainId,
-                to_network: network.chainId,
+                chain_id: hexToDecimal(currentChainId),
+                from_network: hexToDecimal(currentChainId),
```

Comment:
Should we be changing the `from_network` and `to_network` here? I thought the request was only to change `chain_id`?